### DISCUSSION TITLE
The struggle of leveling on Chromiecraft.

### DISCUSSION THREAD

**Discussion by LiaohuChromie** [chromiecraft/chromiecraft#4148] [Suggestions]
Hello, 

First of all, i am not an native english speaker so i might use the wrong word or make some grammar mistake here and there. Shouldn't prevent you to understand what i'm trying to convey though. Apologies in advance.

My suggestion is to increase the XP to make is x2 baseline, and x3 during week-ends. And i'll tell you my reasoning below : 

Having a x1 XP rate was totally fine when the server was in its early progression state, and level brackets were much lower. But now (and for some time) that the progression has reached level 60 and raiding content is open, the leveling content feels empty and terrible to experience.

> The content is blizzlike, there is nothing new to experience, we are doing the same old blizzard's questing content as before. 

> The already low pop is now divided between newcomers / people leveling alts and the people at 60.

> Most people who level multiple alts will just wait for the XP weekend event and spam RDF/BGs, skipping most of the outdoor content and making it even more empty of players during week day. and creating the problem just bellow, who is in direct contraction with the server's purpose.

> People have told me that the x1 rates was here so people take time to level, find bugged content and report it. Except, by having x1 rate : 
- You deter a lot of people from playing here, which would mean more play-tester and more people to find and report bugs.
            
- People are mostly NOT doing the leveling content, they are just waiting for the weekend. So there is even less people experiencing quests than normal. Which mean less bug found, reported and fixed.
            
- By having a baseline x2 XP and x3 during weekend, people would be incentivized to have more alts. Making them experiencing the content mutiple times, with differents races/factions/classes. Also, the incrased XP rate would mean it wouldn't feels as bad to play during week days. 
            
Again, i'm not asking for crazy XP rates such as x7 or even x10 as other server. But a x2 baseline would be a nice quality of life improvement for the playerbase and in my opinion, would be an healthy change for the server overall. 

     Have a good day !

**Comment by 55Honey** [comment]
Hello @LiaohuChromie ,
Thanks for your suggestion.

New players already have 2x XP rates by using the recruit-a-friend(RAF) feature. That goes for new accounts as well. So if a player decides to level one or two more characters later on, they can just create a second account and activate the RAF on it. Up to 5 accounts per player are allowed.

With the weekend XP being x2 and the RAF bonus being x2 already, there's not much room for higher rates currently.

Regards

**Reply by clovaek** [reply]
Personally, I think a full time x2 rate makes more sense than having it only during the weekend. I don't bother leveling alts outside of the weekend because the time is more efficiently spent during that bonus weekend.
